Milk_Jug_Glow_Skull/milk_jug_glow_skull.py

```diff
@@ -0,0 +1,63 @@
+import time
+import board
+import digitalio
+import audioio
+import neopixel
+
+# user variables
+pix_rate = 0.03  # Increase the number to slow down the color chase
+blink_times = 2  # number of times the eyes blink between color chases
+blink_speed = 0.1  # speed of the blinks, lower numbers are faster
+rest_time = 3  # time between color changes e.g. '3' = 3 sec, '300'= 5 mins.
+
+#  setup
+NEOPIXEL_PIN = board.EXTERNAL_NEOPIXEL
+NUM_PIXELS = 30
+pixels = neopixel.NeoPixel(NEOPIXEL_PIN, NUM_PIXELS)
+
+
+led = digitalio.DigitalInOut(board.D13)
+led.direction = digitalio.Direction.OUTPUT
+led.value = True
+time.sleep(0.5)
+
+ORANGE = (255, 30, 0)
+PURPLE = (200, 0, 255)
+RED = (255, 0, 0)
+GREEN = (0, 255, 0)
+BLACK = (0, 0, 0)
+
+COLORS = [ORANGE, PURPLE, RED, GREEN, ORANGE, PURPLE, RED]
+
+pixels.fill(BLACK)
+pixels.show()
+
+def color_chase(color, wait):
+    for i in range(NUM_PIXELS):
+        pixels[i] = color
+        time.sleep(wait)
+        pixels.show()
+    time.sleep(rest_time)
+
+def blink(times, speed):
+    for _j in range(times):
+        led.value = False
+        time.sleep(speed)
+        led.value = True
+        time.sleep(speed)
+
+def play_waves(file_num):
+    wave_file = open(wave_files[file_num], "rb")
+    wave = audioio.WaveFile(wave_file)
+    audio.play(wave)
+
```

Comment:
you never close the file or wait for it to finish?

Reply:
I found you can do some fun things by setting the times very low and it just hits the start of each scream. Is this ok to let it start new ones, interrupting the old ones?

Reply:
:/ its kinda bad code practice, if you wanted to do this have it play the audio, then time.delay() and stop playing the audio before returning

Reply:
Got it. I've added the while `audio.playing: pass` line to fix